synthesizer/process/src/stack/deploy.rs

```diff
@@ -124,12 +124,20 @@ impl<N: Network> Stack<N> {
             lap!(timer, "Compute the request for {}", function.name());
             // Initialize the assignments.
             let assignments = Assignments::<N>::default();
+            // Initialize the constraint limit. Account for the constraint added after synthesis that randomizes vars.
```

Comment:
// Initialize the constraint limit. Account for the constraint added after synthesis that makes the Varuna zerocheck hiding.